website/src/parsers/scala/index.js

```diff
@@ -0,0 +1,6 @@
+import 'codemirror/mode/clike/clike';
+
+export const id = 'text/x-scala';
```

Comment:
`id`s are usually meant to be same as folder name, this looks like was supposed to go to `mimeTypes` instead?